src/components/Dialog.js

```diff
@@ -14,17 +37,5 @@ export default function Dialog(type, options) {
     }
   }
 
-  if (type == 'Open') {
-    return libui.UiDialogs.openFile(CURRENT_WINDOW);
-  } else if (type == 'Save') {
-    return libui.UiDialogs.saveFile(CURRENT_WINDOW);
-  } else if (type == 'Message') {
-    libui.UiDialogs.msgBox(CURRENT_WINDOW, options.title, options.description);
-  } else if (type == 'Error') {
-    libui.UiDialogs.msgBoxError(
-      CURRENT_WINDOW,
-      options.title,
-      options.description
-    );
-  }
+  getCurrentDialog[type](options);
```

Comment:
This should be a `return` @kusti8 ?

Reply:
Yes

Reply:
Otherwise the file names won't get returned to the user.